<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ceasor001" failures="1" tests="1" name="Subscription Test" time="5.488" errors="0" timestamp="2024-10-09T11:15:32 IST">
  <testcase classname="com.demoshop.test.SubscriptionTest" name="verifyUserIsAbleToSubscribeWithValidEmail" time="0.776">
    <failure type="java.lang.AssertionError" message="expected [Thank you for signing up! A verification email has been sent. We appreciate your interest.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Thank you for signing up! A verification email has been sent. We appreciate your interest.] but found []
at com.demoshop.test.SubscriptionTest.verifyUserIsAbleToSubscribeWithValidEmail(SubscriptionTest.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserIsAbleToSubscribeWithValidEmail -->
</testsuite> <!-- Subscription Test -->
